backend/core/src/applications/types/application-multiselect-question-option.ts

```diff
@@ -19,6 +19,18 @@ export class ApplicationMultiselectQuestionOption {
   @ApiProperty()
   checked: boolean
 
+  @Expose()
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @ApiProperty({ required: false })
+  addressHolderName?: string
+
+  @Expose()
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @ApiProperty({ required: false })
```

Comment:
It works to put this data at the option level. But I think this data might be better in the "extraData" array field below since it is more dynamic. You could use "addressHolderRelationship" and "addressHolderName" as the extraData keys and type of "TextInput"

Reply:
@ludtkemorgan  It make sense, but extraData behaves strangely atm. First of all to extract address from it we either rely on it being first item on array (which rather should be still true after putting there more items) or by testing it's type. Keys from extraData objects for some reason are overwritten by parent `key` (so instead for example `address` its `work in county`) I believe that i can change that, assuming that the key is redundant for address.

Reply:
I think we can get away with changing the key of the extra data object to something that is more appropriate. So in your example the "extraData" object of type "address" will also have the key of "address" instead of "work in county". I don't think we use that key anywhere at the moment so it should be ok unless the frontend is using it somewhere I can't find.